Arguments: 
  /Users/s/.nvm/versions/node/v16.13.1/bin/node /Users/s/.nvm/versions/node/v16.13.1/bin/yarn add node 18.4.0

PATH: 
  /Users/s/.nvm/versions/node/v16.13.1/bin:/Users/s/.nvm/versions/node/v19.4.0/bin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/usr/local/share/dotnet:~/.dotnet/tools:/Users/s/.nvm/versions/node/v16.13.1/bin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/s/go/bin:/Users/s/go/bin

Yarn version: 
  1.22.18

Node version: 
  16.13.1

Platform: 
  darwin arm64

Trace: 
  Error: https://registry.yarnpkg.com/18.4.0: Not found
      at Request.params.callback [as _callback] (/Users/s/.nvm/versions/node/v16.13.1/lib/node_modules/yarn/lib/cli.js:66138:18)
      at Request.self.callback (/Users/s/.nvm/versions/node/v16.13.1/lib/node_modules/yarn/lib/cli.js:140883:22)
      at Request.emit (node:events:390:28)
      at Request.<anonymous> (/Users/s/.nvm/versions/node/v16.13.1/lib/node_modules/yarn/lib/cli.js:141855:10)
      at Request.emit (node:events:390:28)
      at IncomingMessage.<anonymous> (/Users/s/.nvm/versions/node/v16.13.1/lib/node_modules/yarn/lib/cli.js:141777:12)
      at Object.onceWrapper (node:events:509:28)
      at IncomingMessage.emit (node:events:402:35)
      at endReadableNT (node:internal/streams/readable:1343:12)
      at processTicksAndRejections (node:internal/process/task_queues:83:21)

npm manifest: 
  {
    "dependencies": {
      "19": "^0.0.0",
      "arweave": "^1.12.2",
      "openssl": "^2.0.0"
    }
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile
